### DISCUSSION TITLE
blog/retrospect/6-devcourse-MIL-4

### DISCUSSION THREAD

**Discussion by giscus** [pySoo/Next-blog#78] [General]
# blog/retrospect/6-devcourse-MIL-4

프로젝트에서 협업을 잘 하기 위해 시도했던 방법들과 디자인 및 공통 컴포넌트 개발 과정에서 배운 점을 기록하였습니다.

https://enjoydev.life/blog/retrospect/6-devcourse-MIL-4

**Comment by Songhee99** [comment]
수현님 고생 많으셨습니다 !! 맛남의 길 프로젝트에 대해 자세히 알 수 있어 좋네요 ㅎㅎ

**Reply by pySoo** [reply]
앗 송희님!! 😍 여기서 뵙게 되다니 너무 반갑네요 ㅋㅋㅋ
송희님도 한 달 동안 너무 고생 많으셨어요 ㅎㅎ 리팩토링도 화이팅 해봐요! 💪
